packages/scripts/config/webpack.config.js

```diff
@@ -81,7 +33,6 @@ const config = {
 		filename: '[name].js',
 		path: path.resolve( process.cwd(), 'build' ),
 	},
-	externals,
```

Comment:
Out of curiosity, what will happen if someone provides their own list of externals? Will it merge with what plugin provides?

Reply:
This is something I want to better understand myself. I plan to add tests that will clarify the expected behavior.

I suspect that they are compatible, but they may conflict. Proper documentation —another to-do 🙂 — should indicate what to expect and how to deal with issues.

Reply:
I've done my best to document this:

> **Note:** This plugin overlaps with the functionality provided by [webpack
> `externals`](https://webpack.js.org/configuration/externals). This plugin is intended to extract
> script handles from bundle compilation so that a list of script dependencies does not need to be
> manually maintained. If you don't need to extract a list of script dependencies, use the `externals`
> option directly.
> 
> This plugin is compatible with `externals`, but they may conflict. For example, adding
> `{ externals: { '@wordpress/blob': 'wp.blob' } }` to webpack configuration will effectively hide the
> `@wordpress/blob` module from the plugin and it will not be included in dependency lists.

https://github.com/WordPress/gutenberg/blob/1fd0b69b2cade06eb85eb3c77bc0a838d3333375/packages/dependency-extraction-webpack-plugin/README.md#webpack

I've also added some tests around this:

https://github.com/WordPress/gutenberg/blob/1fd0b69b2cade06eb85eb3c77bc0a838d3333375/packages/dependency-extraction-webpack-plugin/test/fixtures/with-externs/webpack.config.js#L1-L7
https://github.com/WordPress/gutenberg/blob/1fd0b69b2cade06eb85eb3c77bc0a838d3333375/packages/dependency-extraction-webpack-plugin/test/fixtures/with-externs/index.js#L1-L18

https://github.com/WordPress/gutenberg/blob/1fd0b69b2cade06eb85eb3c77bc0a838d3333375/packages/dependency-extraction-webpack-plugin/test/__snapshots__/build.js.snap#L44-L73